include/d/actor/d_a_wall.h

```diff
@@ -20,7 +25,27 @@ class daWall_c : public fopAc_ac_c {
     BOOL _draw();
 
 public:
-    /* Place member variables here */
+    /* 0x290 */ request_of_phase_process_class mPhs;
+    /* 0x298 */ J3DModel* mpModel;
+    /* 0x29C */ dCcD_Stts mStts;
+    /* 0x2D8 */ dCcD_Tri mTri[2];
+    /* 0x578 */ dBgW* mpBgW;
+    /* 0x57C */ Mtx mMtx;
+    /* 0x5AC */ u8 mState;
+    /* 0x5B0 */ dPa_smokeEcallBack mSmokeCb;
+    /* 0x5D0 */ JPABaseEmitter* mpEmitter;
+    /* 0x5D4 */ float mpDst;
```

Comment:
```suggestion
    /* 0x5D4 */ f32 mDst;
```

`mp` is for pointer members. this is a float, not a pointer to a float, so it shouldn't have the p.